src/metabase/models/json_migration.clj

```diff
@@ -0,0 +1,25 @@
+(ns metabase.models.json-migration)
+
+(defn update-version
+  "Returns a function that updates the column to the specified version. Useful when `comp`ed together with a JSON
```

Comment:
I think this docstring has drifted. This no longer returns a function

Reply:
Good catch, fixed